axelor-sale/src/main/resources/views/Selects.xml

```diff
@@ -105,4 +105,9 @@
     <option value="2">Product</option>
   </selection>
 
+  <selection name="line.invoicing.mode.select">
+    <option value="1">Directly</option>
+    <option value="2">After delivery</option>
+  </selection>
```

Comment:
The name of the select is not correct. It should be 'business.project.sale.order.line.invoicing.mode.select' to give more information.